python/pyspark/ml/classification.py

```diff
@@ -109,6 +109,10 @@ class LinearSVC(JavaEstimator, HasFeaturesCol, HasLabelCol, HasPredictionCol, Ha
     .. versionadded:: 2.2.0
     """
 
+    threshold = Param(Params._dummy(), "threshold",
+                      "threshold in binary classification prediction applied to rawPrediction",
```

Comment:
This is the only doc that Python users will see, right? Should we clarify here what rawPrediction is like we did for Scala and R?